src/main/resources/db/changelog/changes/create-category-book-table.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-category-book-table
+      author: YEvhenii
+      changes:
+        - createTable:
+            tableName: category_book
+            columns:
+              - column:
+                  name: category_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
add primary keys to both columns and also you can use foreign keys

Reply:
Agree. Add primary key(category_id, book_id) and foreign key constraints